sonar-api-model/build.gradle.kts

```
@@ -0,0 +1,10 @@
+plugins {
+    id("io.codemodder.java-library")
```

this is missing the recommended plugin 
```suggestion
    id("io.codemodder.java-library")
    id("io.codemodder.maven-publish")
```